src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -333,12 +333,10 @@ func (r *StatefulSetResource) obj() (k8sclient.Object, error) {
 							Args: []string{
 								"redpanda",
 								"start",
-								"--check=false",
-								"--smp 1",
 								// sometimes a little bit of memory is consumed by other processes than seastar
```

Comment:
can we remove this too ? what do you think?

Reply:
you mean the reserve memory? There was some issue when that was not set, redpanda refused to start see https://github.com/vectorizedio/redpanda/pull/668

I mean if we want to drop it, we probably need to do some more investigation in that. Michal recommended to add this

Reply:
The https://github.com/vectorizedio/redpanda/pull/668 introduced `--reserve-memory`, because when Redpanda has enabled TLS the process starts to consume more memory than `--memory` flag sets.

Reply:
ack